### DISCUSSION TITLE
Display view range not working

### DISCUSSION THREAD

**Discussion by Thomas7191** [ShaneBeee/SkBee#610] [General]
Hello, I have a question regarding the display view range. 
When I set it to 150 ( set display view range of {_lead} to 150 ), but in the spigot.yml under entity-tracking-range: display: 80, 
why does the server override the 150 and set the display view range to 80?

**Comment by ShaneBeee** [comment]
because the server overrides, you'd have to ask them why that happens.

**Comment by Thomas7191** [comment]
Hello, I fixed it. I set the display view range to 0.3. "set display view range of {_lead} to 0.3". And that resolved my issue.
